zz/zzmaiores.sh

```diff
@@ -68,6 +68,7 @@ zzmaiores ()
 		resultado=$(
 			find $pastas $recursivo -type f -ls |
 				tr -s ' ' |
+				zzlblank |
```

Comment:
Muito obrigado pela sua contribuição @daaugusto, e também pelo descrição bem completa do problema e da solução 🏅

Só um detalhe, para remover espaços em branco, temos a [zztrim](https://github.com/funcoeszz/funcoeszz/blob/master/zz/zztrim.sh), que com a opção `-l` (ou `--left`) remove os espaços à esquerda. Para este caso específico, a `zzlblank` também funciona, mas não é o ideal visto que seu uso é mais relacionado à indentação de textos e códigos.

```suggestion
				zztrim -l |
```